app/views/jobseekers/registrations/new.html.haml

```diff
@@ -2,20 +2,20 @@
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    %h1.govuk-heading-l= t(".title")
-
-    %p.govuk-body= t(".description")
+    %h1.govuk-heading-xl= t(".title")
 
     = form_for resource, url: jobseeker_registration_path do |f|
       = f.govuk_error_summary
 
+      %p.govuk-body= t(".description")
```

Comment:
I don't think the descriptions should be inside the form?

Reply:
See commit message

Reply:
Hmm ok I concede defeat, just looked a bit weird to me!

Reply:
i dont think its against the form gods to have textual elements inside a form except they'd usually be label elements